www/index.html

```diff
@@ -977,7 +977,22 @@
           {{/each image}}
         </div>
       </script>
-  </section>
+
+      <script type="text/template7" id="checkTermsVersionTemplate">
+        <div class="popup popup-terms">
+          <div class="background-overlay"></div>
+           <div class="block">
+             <div class="popup-text">För att fortsätta använda f-appen och vår hemsida måste du acceptera att vi använder din användardata.
```

Comment:
Change the text to match the website
```suggestion
             <div class="popup-text">Du behöver läsa igenom och godkänna den senaste versionen av F-sektionens sekretessvillkor innan du kan fortsätta. Sekretessvillkoren finns att läsa
```